src/openrct2/Game.cpp

```diff
@@ -630,96 +615,6 @@ void game_log_multiplayer_command(int command, const int* eax, const int* ebx, c
         format_string(log_msg, 256, STR_LOG_DEMOLISH_RIDE, args);
         network_append_server_log(log_msg);
     }
-    else if (
-        command == GAME_COMMAND_SET_RIDE_APPEARANCE || command == GAME_COMMAND_SET_RIDE_VEHICLES
```

Comment:
This is not dead code. GAME_COMMAND_SET_RIDE_VEHICLES is still used and GAME_COMMAND_SET_RIDE_SETTING

Reply:
Sorry about that. I've put it back in.

Reply:
No worries, your PR is appreciated.